tools/defaults.bzl

```diff
@@ -30,7 +31,7 @@ def ts_library(name, testonly = False, deps = [], **kwargs):
         devmode_target = "es2020",
         prodmode_module = "commonjs",
         prodmode_target = "es2020",
-        tsconfig = "//tools:tsconfig",
+        tsconfig = kwargs.pop("tsconfig", "//tools:tsconfig"),
```

Comment:
Nit: Can you add a TODO to clean this up? I think we'd not want to use the `tools/tsconfig` ever for code not being in `tools`?